.gitignore

```diff
@@ -5,6 +5,7 @@ vendor/cairo-lang/build
 
 # Secrets
 /.warp-activation-token
+/.warp-deps
```

Comment:
What command generates that?

Reply:
Ah, something I'd added to the makefile a long time ago but seems like someone changed the makefile